backend/wargamebackend/wargamelogic/views/post.py

```
@@ -13,12 +13,17 @@
 from ..game_logic import (
     move_unit_instance
 )
+from ..check_roles import (
+    role_required, any_role_required
```

Ruff (0.12.2)

`..check_roles.role_required` imported but unused

Remove unused import

(F401)